### ISSUE TITLE
Encrypt local content 

### ISSUE THREAD

**Issue by kistitan** [fetlife/android#162]
A possible enhancement for the future to have all local content encrypted and decrypt it only when the user provides her/his password.
This would need some change in the backend too to provide cipher keys.

As it is possible to encrypt the whole device content on most of the Android phones, this issue has low priority.

**Comment by johnbaku** [comment]
I like!

**Comment by bracuphoth** [comment]
Encryption is a buzz word. If encryption is needed, full disk encryption is the most reliable. And maybe having a PIN to open the app would address the issue of privacy.

On the other hand, encryption usually means an extra library installed, with its own bugs. Encryption means higher memory consumption. Higher CPU consumption. Lower battery life. Longer response time from the app and the other apps already running on the phone or tablet.

**Comment by kistitan** [comment]
That is true, on the other hand extra encryption can give some extra protection against malicious apps on routed devices, as they can scan the saved content of other Apps too.
This is on the other hand quite rare usecase at the moment.
Furthermore we are about to significantly reduce the locally saved content, so I decrease the priority.
